packages/web-react/src/components/Dropdown/demo/DropdownPlacements.tsx

```diff
@@ -1,83 +1,81 @@
-import React, { ChangeEvent, Ref, useState } from 'react';
-import { DropdownRenderProps, PlacementDictionaryType } from '../../../types';
+import React, { ChangeEvent, useState } from 'react';
+import { PlacementDictionaryType } from '../../../types';
 import { Button } from '../../Button';
 import { Grid, GridItem } from '../../Grid';
 import { Item } from '../../Item';
 import { Radio } from '../../Radio';
-import { Dropdown } from '..';
+import Dropdown from '../Dropdown';
+import DropdownPopover from '../DropdownPopover';
+import DropdownTrigger from '../DropdownTrigger';
 
 const DropdownPlacements = () => {
-  const [placement, setPlacement] = useState<PlacementDictionaryType>('bottom-start');
+  const [isOpen, setIsOpen] = React.useState(false);
+  const [placement, setPlacement] = useState<PlacementDictionaryType>('bottom-left');
+  const onToggle = () => setIsOpen(!isOpen);
 
   const handlePlacementChange = (e: ChangeEvent<HTMLInputElement>) => {
     setPlacement(e.target.value as PlacementDictionaryType);
   };
 
-  const dropdownTrigger = ({ trigger: { className, ref, ...restOf } }: DropdownRenderProps) => (
-    <Button UNSAFE_className={className} ref={ref as Ref<HTMLButtonElement>} {...restOf}>
-      <span style={{ whiteSpace: 'nowrap' }}>{placement}</span>
-    </Button>
-  );
-
   return (
     <form autoComplete="off">
       <Grid cols={3} marginX="auto" UNSAFE_style={{ alignItems: 'center', justifyItems: 'center', maxWidth: '40rem' }}>
         <GridItem columnStart={2} rowStart={1}>
           <Radio
             name="placement"
-            isChecked={placement === 'top-start'}
+            isChecked={placement === 'top-left'}
             isLabelHidden
             onChange={handlePlacementChange}
-            id="placement-top-start"
-            label="top-start"
-            value="top-start"
+            id="placement_top_left"
+            label="top-left"
+            value="top-left"
```

Comment:
↕️ All these placements should be flow-relative, i.e. they were correct before these changes happened.

```suggestion
            id="placement-top-start"
            label="top-start"
            value="top-start"
```